lib/vmdb/loggers.rb

```diff
@@ -111,9 +110,9 @@ def self.configure_external_loggers
     def self.apply_config_value(config, logger, key)
       old_level      = logger.level
       new_level_name = (config[key] || "INFO").to_s.upcase
-      new_level      = VMDBLogger.const_get(new_level_name)
+      new_level      = ManageIQ::Loggers::Base.const_get(new_level_name)
       if old_level != new_level
-        $log.info("MIQ(#{name}.apply_config) Log level for #{File.basename(logger.filename)} has been changed to [#{new_level_name}]")
+        $log.info("MIQ(#{name}.apply_config) Log level for #{File.basename(logger.logdev.filename)} has been changed to [#{new_level_name}]")
```

Comment:
I think logdev could be nil for particular loggers (particularly those backed by io).  I think Vmdb::Logger.filename handle that.

Reply:
True, though the current code will generate a strange error message if it's nil.

Reply:
@Fryguy updated, and adjusted the message if it's nil.